app/models/pipedrive_client.rb

```diff
@@ -46,6 +46,14 @@ def log(type, message, external_entity_name = nil, external_id = nil)
     raise message
    end
 
+  class << self
+
+    def authorization(email, password)
```

Comment:
If in future we need to add more class methods we can add in them here, instead of writting self with every class method.